Library/Entitlement.cs

```diff
@@ -0,0 +1,105 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Xml;
+
+namespace Recurly
+{
+
+    public class Entitlement : RecurlyEntity
+    {
+        public string AccountCode { get; private set; }
```

Comment:
If Account code isnt supposed to be here lets remove it.  Otherwise we need to set it because its currently null